README.md

```diff
@@ -1 +1,106 @@
-# cairo-annotations
+# Cairo Annotations
+
+The `cairo-annotations` crate provides tools tailored for working with annotations in the Cairo language.
+These
+annotations are part of the Sierra Debug Information format and serve to enrich the code by detailing information.
+
+> 📝 **Note**
+>
+> Although this crate is primarily utilized by projects like `cairo-coverage`, `cairo-profiler`, and `starknet-foundry`,
+> it is also fully capable of functioning as a standalone library to work with Sierra Debug Information annotations.
+
+## Features of Cairo Annotations
+
+### Structured Annotations
+
+`cairo-annotations` offers a structured representation, allowing for more ergonomic manipulation of annotations.
+Some
+key features include:
+
+- **Coverage Annotations**: Track locations in the Cairo code that correspond to specific Sierra statements.
+- **Profiler Annotations**: Provide mappings from Sierra statements to fully qualified Cairo paths, detailing which
+  functions in the Cairo code triggered them.
+
+All annotations implement the `TryFromDebugInfo` trait, enabling their extraction from Sierra debug information.
+Here's
+a simple example:
+
+```rust
+let annotations = VersionedCoverageAnnotations::try_from_debug_info(sierra_debug_info).unwrap();
+```
+
+### Coverage Annotations
+
+Coverage annotations provide a mapping from Sierra statement indices to sources in the Cairo code that resulted in their
+creation.
+For extensive documentation,
+see [CoverageAnnotationsV1](./crates/cairo-annotations/src/annotations/coverage.rs).
```

Comment:
nit: for now it does not matter much, but for the future it would be nice to have an information about versioning * somewhere *.

Reply:
Will add this 🔜